frontend/src/components/ui-kit/components/Input/InputSearch/InputSearch.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, memo } from 'react';
+import styles from './InputSearch.module.scss';
+
+import { ICONS } from '../../../icons';
+import IconButton from '../../IconButton';
+
+const InputSearch = memo(({ search, setSearch }) => {
+  const [smallScreen, setSmallScreen] = useState(false);
+  const [isFocused, setIsFocused] = useState(false);
+
+  useEffect(() => {
```

Comment:
Let's do it more universally, make a resize hook (look at the project structure and add the appropriate folder) and why is the screen value so small?